apps/simple-trading-app-e2e/src/integration/market-selector.test.ts

```diff
@@ -28,7 +28,11 @@ describe('market selector', () => {
         .children()
         .find('[role="button"]')
         .should('contain.text', markets[0].name);
-      cy.getByTestId('market-pane').children().find('[role="button"]').click();
+      cy.getByTestId('market-pane')
+        .first()
```

Comment:
there should be only one `market-pane` there. Giving method `first()` in chain after `cy.getByTestId('market-pane')` seems to me something is not correct there...

Reply:
Maybe there is a case that there are two markets with exactly the same market name. So would it be better then:
```
cy.getByTestId('market-pane').children().find('[role="button"]').first().click();
```
?

Reply:
Fair point, I've changed it to include .first() after finding the button